.rubocop.yml

```diff
@@ -17,3 +17,6 @@ AllCops:
     - 'app/views/examples/**/*'
     - 'app/models/example*'
   NewCops: enable
+Style/Documentation:
```

Comment:
lets not edit the rubocop config or the reek config

Reply:
Moved it back to the original state